erroranalysis/erroranalysis/_internal/error_analyzer.py

```diff
@@ -13,9 +13,11 @@
 from erroranalysis._internal.surrogate_error_tree import (
     compute_error_tree as _compute_error_tree)
 from erroranalysis._internal.error_report import ErrorReport
-from erroranalysis._internal.constants import ModelTask, Metrics
+from erroranalysis._internal.constants import ModelTask, Metrics, MatrixParams
 from erroranalysis._internal.version_checker import check_pandas_version
 
+BIN_THRESHOLD = MatrixParams.BIN_THRESHOLD
```

Comment:
Can `compute_matrix` just use `MatrixParams.BIN_THRESHOLD` directly? Looks like this is a common pattern in this PR. Is this just done to make the code look cleaner when fitting in 80 columns?

Reply:
"Is this just done to make the code look cleaner when fitting in 80 columns?"
yes exactly, it's to reduce line counts since I keep hitting the 79 chars limit all the time